modules/core/src/lib/view-manager.ts

```diff
@@ -29,12 +29,15 @@ import type View from '../views/view';
 import type {Timeline} from '@luma.gl/engine';
 import type {EventManager} from 'mjolnir.js';
 import type {ConstructorOf} from '../types/types';
+import type {AnyViewState} from '../views/types';
+
+export type ViewStateMap = AnyViewState | {[viewId: string]: AnyViewState};
```

Comment:
Does the type refine if a value is provided?

Reply:
ish... it won't match what you provide to `views` if that's the question. But the type will narrow down if you start providing `longitude` or `target`.